projects/igniteui-angular/src/lib/grids/tree-grid/tree-cell.component.ts

```diff
@@ -10,6 +10,7 @@ import { IgxGridSelectionService, IgxGridCRUDService } from '../selection/select
 import { HammerGesturesManager } from '../../core/touch';
 import { GridType } from '../common/grid.interface';
 import { IgxGridExpandableCellComponent } from '../grid/expandable-cell.component';
+import { IgxChipComponent } from '../../chips';
```

Comment:
this file has a lot of unused imports, if we are changing it at all, remove them all, otherwise just revert it to original state